jlm/mlir/backend/JlmToMlirConverter.cpp

```diff
@@ -13,6 +16,9 @@
 #include <llvm/Support/raw_os_ostream.h>
 #include <mlir/IR/Verifier.h>
 
+#include "JlmToMlirConverter.hpp"
+#include <unistd.h>
+
```

Comment:
Why is this header necessary? I do not see any use of it. Could we remove it if it is not used?